### PR TITLE
Add review policy.

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +186,10 @@ Run regular benchmarks with:
 cargo bench --features=bench
 ```
 
+## Reviews
+
+PRs must be reviewed by at least one other collaborator. Once they are reviewed and pass CI, anyone may merge (though it's convention for the author to if they have permissions).
```
Comment (YaLTeR) [inline README.md]: Mention AWCY?

**Comment by rzumer** [comment]
I am a bit uneasy about including URLs in commit messages, since they will probably eventually become inaccessible (and AWCY URLs can be quite long). Why are they needed specifically?

**Comment by YaLTeR** [comment]
Probably enough to have them in the PR

**Comment by tdaede** [comment]
It's so you can jump and look at more detailed results. I could make it optional I suppose.

**Comment by shssoichiro** [comment]
In addition to rebasing, should there be a comment about squashing commits? Do we enforce squashing commits?

**Comment by lu-zero** [comment]
We do not enforce squashing, sometimes makes sense if you push additional
refinements as commits instead of reading and pushing an update.

**Comment by tdaede** [comment]
I originally enforced squashing, everyone hated it so I stopped. The main issue is that CI doesn't run on each individual commit, so by not squashing we have some revs that were never CI tested.

**Comment by YaLTeR** [comment]
On the one hand yes, but on the other hand I just had bisection pinpoint me to the exact small change leading to #1513 and that's super helpful to have.

**Comment by tdaede** [comment]
@lu-zero please re-review

**Comment by barrbrain** [comment]
To clarify, the **Contributing** chapter has moved to [CONTRIBUTING.md](https://github.com/xiph/rav1e/blob/master/CONTRIBUTING.md). This section should be added to that document.

**Comment by barrbrain** [comment]
This might need to be manually reapplied to the new file but it would be great to finally merge this.

**Review by barrbrain — APPROVED** [review]

**Pull request merged**